src/js/time.js

```diff
@@ -0,0 +1,14 @@
+const Time = (function() {
+  const { local, fromObject } = luxon.DateTime;
+
+  return {
+    isContactFormAvailable() {
+      const now = local().setZone('Europe/London');
+      const opening = fromObject({ hour: 9, minute: 30, zone: 'Europe/London' });
+      const closing = fromObject({ hour: 16, minute: 30, zone: 'Europe/London' });
```

Comment:
Could pull that zone string out into a const.

Reply:
Done